python/amici/sbml_import.py

```diff
@@ -1269,7 +1269,7 @@ def replace_assignments(formula: str) -> sp.Basic:
         self.symbols['llhy']['name'] = l2s(llh_y_syms)
         self.symbols['llhy']['identifier'] = llh_y_syms
 
-    def _process_conservation_laws(self, ode_model) -> None:
+    def process_conservation_laws(self, ode_model) -> None:
```

Comment:
Maybe stupid, but: Just want to understand, why not starting with an underscore here...

Reply:
underscore mark private methods/members that are not part of the public API. Generally shouldn't be used outside the scope of the respective functions and may change without backwards-compatibility.

Reply:
okay, and that's because it used across 2 files, you wouldn't underscore it?

Reply:
it's not so much about the files. Its called from an ODE_model method, which is by definition not a `SbmlImporter` method.